package.json

```diff
@@ -49,6 +49,7 @@
         "editor:update-tsd": "pnpm packages:build && node frontend/editor-update-tsd.mjs",
         "prettier": "prettier --write \"./**/*.{js,mjs,ts,tsx,json,yaml,yml,css,scss}\"",
         "prettier:check": "prettier --check \"frontend/**/*.{js,mjs,ts,tsx,json,yaml,yml,css,scss}\"",
+        "prettier:file": "prettier --write",
```

Comment:
I think it's probably easier to write `prettier --write` directly, as nobody else will remember or know of this command, but 🤷

Reply:
It needs to be a package.json script, otherwise prettier won't be on the path. The way I usually do this would be to rename the `prettier` script to `prettier:fix`, leaving `prettier` to just be the raw tool. Then you can do `yarn run prettier --write some/file.txt`. I didn't do this because I didn't want to touch more things in this PR

Reply:
`npx prettier --write` works well in case your path doesn't automatically include the local scripts.

Reply:
You don't need to this in either `pnpm` or `yarn`, if you run `pnpm prettier`, it's exactly as if `prettier` was in `$PATH`. In this case `pnpm prettier --write` should 100% be equivalent to `pnpm prettier:file`.

Reply:
That's normally true, but if you define a package script `prettier` (which we do on line 50 of package.json) then that takes precendence over just running `prettier` from `.bin`.

Just tested this
<img width="884" alt="Screenshot 2023-12-12 at 10 06 21" src="https://github.com/PostHog/posthog/assets/2056078/0c897d28-eda5-46ea-9e0a-ff9643e199fa">
<img width="880" alt="Screenshot 2023-12-12 at 10 06 51" src="https://github.com/PostHog/posthog/assets/2056078/6bbb82b7-c9f7-4541-87fd-f30e0ac927f8">

`npx` works, I don't usually like using it like this because by default it will install any missing dependencies, meaning that if you do have a bug with your dependencies it can make it harder to track down.